src/Extensions/Abstractions/EventSources/EventSourcesEventIds.cs

```diff
@@ -70,5 +70,15 @@ public enum EventSourcesEventIds
 		/// Event Id for logging Analytics information
 		/// </summary>
 		LogAnalytics = 15,
+
+		/// <summary>
+		/// Event Id for generic host build succeded
+		/// </summary>
+		GenericHostBuildSucceded = 16,
+
+		/// <summary>
+		/// Event Id for generic host build failed
+		/// </summary>
+		GenericHostBuildFailed = 17
```

Comment:
You don't need to specify the numbers. C# auto increments each Enum type. However it does start at 0 not 1

Reply:
Yes, I understand, enum numbers here explicitly to emphasize their importance since changing numbers will break our log aggregation system. Also if you don't specify numbers then they could change after reordering names that are easy to do and would be a breaking change that is hard to notice.

Reply:
Yes you don't want to make changes with the numbers.